.github/workflows/api.yml

```diff
@@ -16,8 +16,10 @@ jobs:
         uses: ./.github/actions/install
         with:
           os: ubuntu-latest
+      - name: fetch tags
+        run: git fetch --tags
       - name: Update version
-        run: echo -e "${GITHUB_REF_NAME}" > cmd/thyra-server/version.txt
+        run: git describe --tags --abbrev=0 --always > cmd/thyra-server/version.txt
```

Comment:
i propose to add a suffix to the version with the short hash of the current commit
this will allow us to differentiate a release version from a dev version